frontend/src/router.js

```diff
@@ -342,7 +377,7 @@ async function requireCamp(to, from, next) {
   await campFromRoute(to)
     .call({ api: { get: apiStore.get } })
     ._meta.load.then(() => {
-      next({ query: to.query })
+      next()
```

Comment:
In 2d81243 you changed it the other way around.
Why do you change it back now?

Reply:
https://github.com/ecamp/ecamp3/pull/3549#discussion_r1278314085